src/blackdoor/cqbe/node/Node.java

```diff
@@ -39,9 +40,19 @@ private void startServer(int port) {
 		serverThread = new Thread(server);
 		serverThread.start();
 	}
-	
-	private void configureAddressing(L3Address wanAddress){
-		me = wanAddress;
+
+	private void configureAddressing(int port, boolean local) throws Exception {
```

Comment:
I can't see when we would ever be "local"
either that should be removed or lots of comments should be added to the code to explain what that is/means